### PR TITLE
Update hubs_privacy_notice.md

### PR DESCRIPTION
**Pull request by jejohnsonmoz**
Updating to account for Firefox > Mozilla Accounts, and to add standard wording re: personal data requests at the end. 

Also updated to change name from Twitter > X

### CODE REVIEW

File: en/hubs_privacy_notice.md [hunk 1]
```diff
@@ -38,11 +38,11 @@ __Attribution information:__ When someone publishes a scene or avatar to Hubs, t
...
+__Payment information.__ When you subscribe to the Services, you will send payment through one of our third-party payment providers: Stripe, Apple, PayPal, or Google Pay. Mozilla receives a record of your account (including your billing address and the last four digits of your payment method) and the status of your account’s subscription, but does not store your full payment details. The payment providers may also collect personal transactional data or device information via cookies and similar technologies to operate and improve their service. You can learn more about how these companies use and process your data by reviewing their Privacy Notices, which are available on each of their respective websites.
 
 __Usage Information.__ We receive information about the rooms and users in your Hubs instance. For instance, we learn when you create and delete rooms, how many rooms are in your Hubs instance, and how many users are in each room. We also receive information about how much storage space your Hubs instance is using.
```
Comment (peiying2) [inline en/hubs_privacy_notice.md:47]: Lower case `i` as in `Usage information.`

File: en/hubs_privacy_notice.md [hunk 2]
```diff
@@ -59,6 +59,16 @@ __Search providers:__ You can search for images, GIFs, and 3D Models to share in
...
 
 __Discord:__ If you connect Discord to Hubs, we store access tokens and the server and channel IDs that have been connected. We will synchronize chat messages, room changes, 2D and 3D objects you create, and whether you join or leave with the connected Discord channel. Hubs does not log any synchronized messages. You can see [Discord’s Privacy Policy](https://discordapp.com/privacy) for more information.
+
+## Other Information You Should Know
+
+You can learn more about [managing your Mozilla Account data](https://support.mozilla.org/kb/firefox-accounts-managing-account-data). Also, much of the information that we store about our Mozilla account users is easily accessible by [signing in](https://accounts.firefox.com/signin) to your account, where you can also update your [data sharing settings](https://accounts.firefox.com/settings/). To make requests regarding your personal data, please contact us through our [Data Subject Access Request Portal](https://privacyportal.onetrust.com/webform/1350748f-7139-405c-8188-22740b3b5587/4ba08202-2ede-4934-a89e-f0b0870f95f0).
```
Comment (peiying2) [inline en/hubs_privacy_notice.md:68]: Lower case `a` as in Mozilla account`

**Review by peiying2 — CHANGES_REQUESTED** [review]
Minor change request.

File: en/hubs_privacy_notice.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
 # Hubs Privacy Notice
-Version 1.1, effective March 21, 2023
-{: datetime="2023-03-21" }
+Version 1.2, effective November 1, 2023
+{: datetime="2023-11-1" }
```
Comment (flodolo) [inline en/hubs_privacy_notice.md:3]:
```suggestion
{: datetime="2023-11-01" }
```

**Review by peiying2 — APPROVED** [review]

**Pull request merged**